devutils/update_submodule.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+UNGOOGLED_REPO=$(dirname $(dirname $(readlink -f ${BASH_SOURCE[0]})))/ungoogled-chromium
+
+CHROMIUM_VERSION=$(curl -sL https://raw.githubusercontent.com/Eloston/ungoogled-chromium/master/chromium_version.txt)
+UNGOOGLED_REVISION=$(curl -sL https://raw.githubusercontent.com/Eloston/ungoogled-chromium/master/revision.txt)
+
+if [ UNGOOGLED_REVISION != "0" ]
+then
+    UPDATED_TAG="${CHROMIUM_VERSION}-${UNGOOGLED_REVISION}"
+else
+    UPDATED_TAG="${CHROMIUM_VERSION}"
+fi
+    
+# Ensure the submodule is initialized.
+git submodule init
+git submodule update
+
+# Update the submodule.
+cd $UNGOOGLED_REPO
+CURRENT_TAG=$(git describe)
```

Comment:
So this will run it in `ungoogled-chromium-archlinux` and compare it against `ungoogled-chromium`? My `git describe` output for `master` of this repo is currently `74.0.3729.108-1-22-gaaff780`; is this supposed to be equivalent to `UPDATED_TAG` at some point?

Reply:
That part of the script will cd to the submodule and get its tag. The part afterwards will compare that tag and `UPDATED_TAG` (you already know how that one is obtained).

Reply:
Oops, yeah that's fine.

Reply:
But wait, when does `74.0.3729.108-1-22-gaaff780` ever equal `UPDATE_TAG`?